app/assets/v2/css/submit_bounty.css

```diff
@@ -80,6 +80,19 @@ textarea:read-only {
   height: 37px;
 }
 
+#gitcoin-tip-section .form__amount-wrapper {
+  width: 33%;
+}
+
+#gitcoin-tip-section .form__flex-group {
```

Comment:
You can instead just use the classes `justify-content-between` in the html element if this is the only style you want.

Reply:
I can do that if you want, but what happens when there are more elements within `#gitcoin-tip-section` and `#pricing-section` that have the `.form__flex-group` class? This class allows those parent elements to have `justify-content: space-between` to be applied to any `.form__flex-group` classes.

Reply:
Hmm I'm good with this for now ^_^